flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -168,6 +169,12 @@ class AndroidFlutterLocalNotificationsPlugin
         return Future.error('method not defined');
     }
   }
+
+  /// Initializes channel for android API >= 26(from Oreo).This is necessary to 
+  /// receive the first message from the background when the channel is not initialized.
+  Future<bool> createNotificationChannel(AndroidNotificationChannel notificationDetails) async {
```

Comment:
Can you update the example app and a button where it will only call this method to create a notification channel?